preferences.yaml

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: >-
+    The API lets you manage attributes associated with users (user preferences).
+
+    You can get any attribute for any user (read only) according to the rights
+    you owe on the user resource. 
+
+    There rights are set up at moment you've subscribed the service exposed by
+    this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_number}/preferences':
```

Comment:
why not just `/users/{fiscal_number}` ?

`preferences` is not an actual resource since you cannot access each `preference` independently, if you had say `GET /users/{fiscal_number}/preferences/{preference_id}` then it would make sense, in this case the preferences are just attribute of the `User` resource